chart/templates/_helpers.tpl

```diff
@@ -77,8 +77,16 @@ Usage:
 {{ include "base_pull_secrets" . }}
 */}}
 {{- define "base_pull_secrets" -}}
-    {{- if .Values.base.imagePullSecrets.enabled }}
-    {{- include "render" (dict "value" .Values.base.imagePullSecrets.secrets "context" $) | nindent 8 }}
+    {{- if .Values.base.imagePullSecrets }}
+        {{- if .Values.base.imagePullSecrets.enabled }}
+        {{- include "render" (dict "value" .Values.base.imagePullSecrets.secrets "context" $) | nindent 8 }}
+        {{- end }}
+    {{- else -}}
+        {{- if .Values.image.pullSecrets }}
+            {{- if .Values.image.pullSecrets.enabled }}
```

Comment:
I think this would work?
```suggestion
        {{- if (.Values.image.pullSecrets).enabled }}
```

Reply:
Helm doesn't seem to like that shortcut:
helm template mayastor .
Error: template: mayastor/templates/mayastor/agents/ha/ha-node-daemonset.yaml:33:12: executing "mayastor/templates/mayastor/agents/ha/ha-node-daemonset.yaml" at <include "base_pull_secrets" .>: error calling include: template: mayastor/templates/_helpers.tpl:80:18: executing "base_pull_secrets" at <.Values.base.imagePullSecrets.enabled>: nil pointer evaluating interface {}.enabled